README.md

```diff
@@ -34,51 +34,72 @@ In order to see the nice font as we used in the demo, add the following rule in
 ```
 
 ## Workshop steps
-- step 0: install Meteor and test locally
+- step 0:
+  - install Meteor and test locally (TODO: link toevoegen naar meteor of anders de stappen)
+  - follow the instructions and run it.
+  - change some things in the code and see what happens
 - step 1: setup the Tic tac toe project
+  - copy the content of the css in the css file generated in your folder
+  - Notice that you don't have to update your browser. Meteor does that for you automagically
   - you can rename the template in your `.html` file, for example `<template name="gameboard"></template>` instead of the `hello` template
-  - begin with `<div class="gameboard"></div>` (Now you should see a nice green schoolboard)
-  - you can add `<div class="box"></div>` into the `.gameboard` in your template
+  - begin with `<div class="gameboard"></div>` in this template(Now you should see a nice green schoolboard)
+  - you can add `<div class="box"></div>` into the `gameboard` div
+  - Tip: with F12 you can open the developers tools of your browser. And if you use console.log (TODO: Link to console.log) in the js file you can print to the console.
 - step 2: display the grid (3 x 3)
-    - you can use for example, [Mongo Collection](http://docs.meteor.com/#/full/mongo_collection)
-    - you can follow the steps explained in [Meteor tutorial](https://www.meteor.com/tutorials/blaze/templates). Instead of tasks from the tutorial, we want to make boxes
-    - you probably want to use a loop in order to create a scalable application, like this:
-    ```javascript
-    for(var i = 0; i < CellsCount; i++){}
+  - there is only one box on our gameboard. For tictactoe we need 9 of them. But we don't want to write 9 divs in our code.
+  - you probably want to use a loop in order to create a scalable application, like this:
+  ```javascript
+  for(var i = 0; i < CellsCount; i++){}
```

Comment:
Step 2 is only to display Cells in console.

Step 3 will be to display Cells in template.